src/sardana/spock/ipython_01_00/genutils.py

```diff
@@ -1280,8 +1288,16 @@ def mainloop(app=None, user_ns=None):
 
 def prepare_input_handler():
     # initialize input handler as soon as possible
-    import sardana.spock.inputhandler
-    _ = sardana.spock.inputhandler.InputHandler()
+
+    from sardana import sardanacustomsettings
+
+    if sardanacustomsettings.SPOCK_INPUT_HANDLER == "Qt":
```

Comment:
```suggestion
    if getattr(sardanacustomsettings, "SPOCK_INPUT_HANDLER", "CLI") == "Qt":
```

Reply:
This is more robust in case someone removed the setting. We try to access to the `sardanacustomsettings` in this way.